apps/web/src/app/[lang]/dho/[id]/@aside/[tab]/space-configuration/page.tsx

```diff
@@ -64,7 +64,37 @@ export default function SpaceConfiguration() {
             name: person?.name,
             surname: person?.surname,
           }}
-          onSubmit={updateSpace}
+          onSubmit={async (updatedSpace, organisationSpaces) => {
```

Comment:
@DSanich not critical, but can we put this in a separate function and call something like onSubmit(updatedSpace, organisationSpaces) on the template?